### ISSUE TITLE
trainer fails with "Could not find option 'output_filename'"

### ISSUE THREAD

**Issue by gobetti** [fastlane/fastlane#19826]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description
I'm migrating to fastlane 2.201.0 which is adding `trainer` and thus requiring not to install it as a Plugin anymore.
However it looks like the trainer facade exposed via Fastlane is different, because I'm getting this error when I call it:
```
Could not find option 'output_filename' in the list of available options: path, extension, output_directory, fail_build, xcpretty_naming, silent
```
and I'm not specifying that `output_filename` option at all, however I do see mentions to it in #19629

##### Command executed
in my Fastfile:
```
    scan(
      output_types: '',
      output_directory: output_directory,
      result_bundle: true,
      fail_build: false
    )
    trainer(
      path: output_directory,
      output_directory: output_directory,
      fail_build: false
    )
```

##### Complete output when running fastlane, including the stack trace and command used
It doesn't seem relevant here, just regular output from `scan` then the already mentioned error from trainer.

### Environment
It doesn't seem relevant here, any environment should be able to reproduce this, but to make this easier I created a repository https://github.com/gobetti/fastlane-201-issues where you can use the branch `trainer-output-filename`. Or this commit https://github.com/gobetti/fastlane-201-issues/commit/fd890253c4412f44d6e865c8e7b078d80c04b0c5
That repository contains a vanilla Xcode 13 project, my only change to it was to add a failing unit test.

**Comment by gobetti** [comment]
note I may be able to rely on scan's internal call to trainer instead of calling it myself if #19827 is resolved

**Comment by joshdholtz** [comment]
I will look into tonight! Thank you for reporting 😁

**Comment by joshdholtz** [comment]
@gobetti I have a fix for this in PR #19828 💪 I'll have this in a release later tonight!
